features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/AuthManager.js

```diff
@@ -177,9 +177,10 @@ class AuthManager {
     logout() {
         let authHeader = this.bearer + AuthManager.getUser().getPartialToken();
         //TODO Will have to change the logout end point url to contain the app context(i.e. publisher/store, etc.)
-        let url = this.host + "/login/logout";
+        let url = this.host + "/login/logout/publisher";
```

Comment:
Created issue https://github.com/wso2/carbon-apimgt/issues/5087 to track removal of hard coded app contexts.